src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -16,9 +16,14 @@
 package com.bmuschko.gradle.docker.tasks.container
 
 class DockerWaitContainer extends DockerExistingContainer {
+
+    int exitCode
```

Comment:
This should not be a free-standing property of the task. Otherwise people might think it's some sort of input properties. Needs to be exposed as extra method like [here](https://github.com/bmuschko/gradle-docker-plugin/blob/master/src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy#L71-L73).